### DISCUSSION TITLE
Next js App Directory Internationalization default locale set

### DISCUSSION THREAD

**Discussion by AwaisTariq18** [vercel/next.js#46811] [Help]
### Summary

Let's say I want to create a  website in en, fr etc.

Based on the [current docs](https://beta.nextjs.org/docs/guides/internationalization), I  created a [lang] directly beneath /app directory.

This means that my URLs would have the prefix, even for the default locale.

Let's say my default locale is en.

I want my URLs to be example.com and example.com/blog for en, and example.com/fr and example.com/fr/blog.
But in this case, my URLs for the en website would be example.com/en and example.com/en/blog. 
how can i make it to that whenever the locale is en (which is default) the url for all the pages will be example.com/home , example.com/blog etc and for all the other locales it will be example.com/fr/blog

**Comment by wwwdepot** [comment]
The beta docs is using "redirect", but you can modify it to use "rewrite"

see other i18n example: https://github.com/vercel/examples/tree/main/edge-middleware/i18n

**Reply by AwaisTariq18** [reply]
@wwwdepot thank you for replying. i see in the example they are not using app directory and the structure is bit different and in my case i basically wants to call pages under [lang] directory to like that example.com/home instead of example.com/en/home (en is just a prefix to get into the lang directory to find pages so inshort i want to hide that en)
